packages/client/src/components/form/ReviewActionComponent.tsx

```diff
@@ -388,35 +388,40 @@ const ReviewActionComponent = ({
                 {intl.formatMessage(buttonMessages.sendForApproval)}
               </Button>
             </ProtectedComponent>
-          ) : !completeDeclaration ? (
-            <ProtectedComponent scopes={[SCOPES.RECORD_SUBMIT_INCOMPLETE]}>
-              <Button
-                type="primary"
-                size="large"
-                id="submit_incomplete"
-                onClick={toggleSubmitModalOpen}
-                disabled={hasErrorsOnFields || totalFileSizeExceeded}
-              >
-                <Upload />
-                {intl.formatMessage(buttonMessages.sendIncomplete)}
-              </Button>
-            </ProtectedComponent>
           ) : (
-            <ProtectedComponent
-              scopes={[SCOPES.RECORD_SUBMIT_FOR_REVIEW]}
-              denyScopes={[SCOPES.RECORD_SUBMIT_INCOMPLETE]}
-            >
-              <Button
-                type="primary"
-                size="large"
-                id="submit_for_review"
-                onClick={toggleSubmitModalOpen}
-                disabled={hasErrorsOnFields || totalFileSizeExceeded}
-              >
-                <Upload />
-                {intl.formatMessage(buttonMessages.sendForReview)}
-              </Button>
-            </ProtectedComponent>
+            <>
+              {!completeDeclaration && (
+                <ProtectedComponent scopes={[SCOPES.RECORD_SUBMIT_INCOMPLETE]}>
+                  <Button
+                    type="primary"
+                    size="large"
+                    id="submit_incomplete"
+                    onClick={toggleSubmitModalOpen}
+                    disabled={hasErrorsOnFields || totalFileSizeExceeded}
+                  >
+                    <Upload />
+                    {intl.formatMessage(buttonMessages.sendIncomplete)}
```

Comment:
If we can update this [message](https://github.com/opencrvs/opencrvs-countryconfig/blob/develop/src/translations/client.csv#L123) or remove this if we don't want to use it.